resolwe_bio/processes/jbrowse/variants.yml

```diff
@@ -0,0 +1,39 @@
+# =========================
+# JBrowse tracks - Variants
+# =========================
+#
+---
+
+- slug: jbrowse-vcf-track
```

Comment:
Why can't we display the `data:variants:vcf` directly in JBrowse? If the JBrowse can display variants directly, wouldn't it be better to support it without a special track? The same way we support BAM data objects directly.